riot/cli.py

```diff
@@ -53,13 +56,23 @@ def convert(self, value, param, ctx):
 )
 @click.option("-v", "--verbose", "log_level", flag_value=logging.INFO)
 @click.option("-d", "--debug", "log_level", flag_value=logging.DEBUG)
+@click.option("-P", "--pipe", "pipe_mode", is_flag=True, default=False)
```

Comment:
Is there a doc option? We should start documenting our flags/options to help usability

Reply:
Good point. It looks like `click.option` accepts a `help` argument. Given that this is not used with other options, perhaps we could leave this as a follow-up task?

Reply:
I think it makes sense to add it here still for `pipe` and then do the others as follow ups